doc/roadmap/v1.0/bareflank_hypervisor_1.0_roadmap.md

```diff
@@ -0,0 +1,130 @@
+# Bareflank Hypervisor 1.0 Roadmap
+
+## Summary
+
+Version 1.0 of the Bareflank hypervisor will provide the project with basic, type 2 support for launching an Intel based VT-x hypervisor. Although the hypervisor will only be capable of launching the hypervisor, and continue execution of the host operating system, this first version will lay the ground work for future versions. 
+
+## Goals
+
+* Provide basic type 2 hypervisor with support for Windows and Linux. 
+* Hypervisor compiled as a separate, cross-compiled ELF binary
+* Support for dynamically loading modules within the hypervisor
+* Basic debugging facilities
+* Basic VT-x exit handler
+* No outside dependencies
+* Hypervisor written in C++
+* Initial support for Intel’s x86_64. 
+
+## Compilation
+
+Although support for clang / LLVM would be preferred, version 1.0 will be compiled using a GCC cross-compiler to simplify the build process. 
+
+* CMake will be used to generate a platform independent build system
```

Comment:
Ordinary GNU Make will be used, as CMake isn't well suited to building projects in such a stripped down environment.